hc/settings.py

```diff
@@ -107,7 +107,7 @@
 
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
+SITE_ROOT = "https://healthchecks-akira.herokuapp.com"
```

Comment:
Make sure when running locally, you override this in your local_settings @Gathondu @daisyndungu @Andela-Didacus @NgangaMaryanne